.changeset/ten-cougars-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
Since astro < 1.0 and this is a non-breaking change this should be a patch.

Reply:
Alright! I will make the changes.